scripts/kind-e2e/e2e.sh

```diff
@@ -311,25 +323,41 @@ if [[ $1 != keep ]]; then
     trap cleanup EXIT
 fi
 
-echo Starting with status: $1, k8s_version: $2, logging: $3, kubefed: $4.
+echo Starting with status: $1, k8s_version: $2, logging: $3, kubefed: $4, broker :$5.
 PRJ_ROOT=$(git rev-parse --show-toplevel)
 mkdir -p ${PRJ_ROOT}/output/kind-config/dapper/ ${PRJ_ROOT}/output/kind-config/local-dev/
 SUBMARINER_BROKER_NS=submariner-k8s-broker
 SUBMARINER_PSK=$(cat /dev/urandom | LC_CTYPE=C tr -dc 'a-zA-Z0-9' | fold -w 64 | head -n 1)
 KUBEFED_NS=kube-federation-system
 export KUBECONFIG=$(echo ${PRJ_ROOT}/output/kind-config/dapper/kind-config-cluster{1..3} | sed 's/ /:/g')
 
-kind_clusters "$@"
-setup_custom_cni
+broker_cluster=cluster1
+if [[ $5 = cluster1 || $5 = cluster2 || $5 = cluster3 ]]; then
+    echo Broker on $5
+    broker_cluster=$5
+elif [[ -n $5 ]]; then
+    echo Invalid broker setting $5, ignoring
```

Comment:
Or may "echo broker cluster not specified. Creating 3 clusters setup with seperate broker cluster"

Reply:
This message is printed if the user asks to install the broker on a non-existing cluster; it’s not the default three-node cluster case.